ddtrace/opentelemetry/tracer.go

```diff
@@ -62,6 +62,30 @@ func (t *oteltracer) Start(ctx context.Context, spanName string, opts ...oteltra
 			o(&cfg)
 		}
 	}
+	// Add span links to underlying Datadog span
+	ddLinks := make([]ddtrace.SpanLink, 0, len(ssConfig.Links()))
```

Comment:
nit: just call this `links` instead of `ddLinks` unless there is a compelling reason to do so.

Reply:
I was trying to distingusih between the otel SpanLink struct and the Datadog SpanLink struct. I will clean this up.